bundle/phases/load.go

```diff
@@ -0,0 +1,29 @@
+package phases
+
+import (
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config/mutator"
+)
+
+// The load phase loads configuration from disk and performs
+// lightweight preprocessing (anything that can be done without network I/O).
+func Load() bundle.Mutator {
```

Comment:
Shall we use it in `loadBundle` as well? https://github.com/databricks/cli/blob/main/cmd/root/bundle.go#L77

Reply:
Great catch. I'm working on a change on top of this one that modifies `MustConfigureBundle` and `TryConfigureBundle` to return diagnostics as well and this is addressed there.